integration-tests/src/test/java/uk/gov/di/authentication/api/AuthorisationIntegrationTest.java

```diff
@@ -57,9 +60,13 @@
     private static final KeyPair KEY_PAIR = KeyPairHelper.GENERATE_RSA_KEY_PAIR();
     public static final String DUMMY_CLIENT_SESSION_ID = "456";
 
+    @RegisterExtension
+    public static final RequestURILambdaStubExtension requestURILambdaStub =
+            new RequestURILambdaStubExtension(6001);
```

Comment:
We don't have to do this now, but to avoid any port conflict, you could not assign a port here and just use a dynamically assigned one. You'd have to override the configuration service like we do here: https://github.com/alphagov/di-authentication-api/blob/8361ab0613f1bc9bfd7ecde026c03999348de72e/shared-test/src/main/java/uk/gov/di/authentication/sharedtest/basetest/NotifyIntegrationTest.java#L23

Reply:
Good shout. I have updated